sdks/go/pkg/beam/io/synthetic/source_test.go

```diff
@@ -175,8 +175,8 @@ func TestSourceConfigBuilder_NumHotKeys(t *testing.T) {
 		{elms: 15, hotKeys: 2, hotKeyFraction: 1.0},
 		{elms: 30, hotKeys: 10, hotKeyFraction: 1.0},
 		{elms: 50, hotKeys: 25, hotKeyFraction: 1.0},
-		{elms: 30, hotKeys: 10, hotKeyFraction: 0.5},
-		{elms: 50, hotKeys: 25, hotKeyFraction: 0.75},
+		// {elms: 30, hotKeys: 10, hotKeyFraction: 0.5}, tests for hotKeyFraction < 1.0 could be Flaky
```

Comment:
In general, we shouldn't commit commented out code. That being said, there are two solutions:
1) remove this completely,
2) if you feel that `hotKeyFraction` should be tested, but not necessarily now for whatever reason, create a // TODO(BEAM-xxx) and describe what should be done here.

Reply:
I got it, I removed Hot Key Fraction testing completely.